superset/views/core.py

```diff
@@ -1330,11 +1335,32 @@ def testconn(self):
             with closing(engine.connect()) as conn:
                 conn.scalar(select([1]))
                 return json_success('"OK"')
-        except Exception as e:
-            logger.exception(e)
+        except NoSuchModuleError as e:
+            logger.info(f"Invalid driver {e}")
```

Comment:
I think the convention is to avoid f-strings for logging: https://docs.python.org/3/howto/logging.html#optimization